src/ChannelServer/Scripting/ScriptManager.cs

```diff
@@ -122,6 +122,7 @@ private void InitializeLua()
 			Register(resetstats);
 			Register(changehair);
 			Register(spawn);
+            Register(levelup);
```

Comment:
Please set your IDE to use tabs for indention, as outlined in our coding conventions, which can be found in the [readme](https://github.com/aura-project/melia#coding-conventions).